script.js

```diff
@@ -21,41 +53,18 @@ var main = function (input) {
   if (
     (player == scissors && computer == paper) ||
     (player == stone && computer == scissors) ||
-    (player == paper && computer == stone)
+    (player == paper && computer == stone) ||
+    (player == reversedPaper && computer == scissors) ||
+    (player == reversedStone && computer == paper) ||
+    (player == reversedScissors && computer == stone)
   ) {
     myOutputValue = `You have played ${input} and you have won the computer's choice ${computer}! <br> good job! <br> Now you can type "scissors, paper or stone to start new game <br>`;
   }
 
   //check for draw
-  if (
-    (player == scissors && computer == scissors) ||
-    (player == stone && computer == stone) ||
-    (player == paper && computer == paper)
-  ) {
-    myOutputValue = `You have played ${input} and you have won the computer's choice ${computer}! <br> Its a draw!!`;
+  if (player == computer) {
+    myOutputValue = `You have played ${input} and its the same as the computer's choice ${computer}! <br> Its a draw!!`;
```

Comment:
[Comment]: Might have missed out on the player = reversed stone and computer = stone scenario.

Reply:
Hi eric, i was wondering because the code is player ==  computer, i do not have to list down all the choices?